subcommands/version/tuf_repo.go

```diff
@@ -0,0 +1,10 @@
+package version
+
+import _ "embed"
+
+// This is generated with:
+//  git show tuf-metadata:repository/root.json | json_pp > ./subcommands/version/root.json
+//go:embed root.json
+var RootContent []byte
```

Comment:
Maybe, `InitialRootContent` or `PreloadedTufRoot`, or any combination of these 2 choices?
That'd make it clear _how_ this variable is used.

Also, it'd be interesting to try making this a `const`, unless you want to override this value in tests.

Reply:
you can't do consts with go:embed but - 6a17a12450a830ddde791ba680263fb8ceb16c9f